### ISSUE TITLE
Proposal: org-mode style of expanding/collapsing

### ISSUE THREAD

**Issue by huyz** [xavierog/moulti#20]
I really like org-mode's TAB and SHIFT+TAB for expanding/collapsing items or the entire tree.

What do you think?

**Comment by xavierog** [comment]
I am not familiar with org-mode. Could you explain how expanding/collapsing works in org-mode?

**Comment by xavierog** [comment]
@huyz Are you still interested in that enhancement suggestion?

**Comment by huyz** [comment]
Yes, just been very busy. Let me get back to you, hopefully with a screencast clip

**Comment by huyz** [comment]
@xavierog Here's the demo: https://youtu.be/GEdOj0zcDWk

Thanks for your patience
